lang/en/docs/tutorials/dft/electronic/hubbard.md

```diff
@@ -0,0 +1,47 @@
+# DFT+U and Hubbard parameter calculation in Quantum Espresso
+
+The electronic structure of materials consisting of transition metals (which
```

Comment:
I'd omit this paragraph - let's just be factual about HOW to execute this instead and leave the explanation of WHY outside of the documentation. Otherwise, there's a lot more to be written regarding when DFT+U should and should not be used.